src/main/java/com/intive/patronage/toz/model/db/Identifiable.java

```
@@ -0,0 +1,23 @@
+package com.intive.patronage.toz.model.db;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import java.util.UUID;
+
+@MappedSuperclass
+public abstract class Identifiable {
+
+    @Id
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+    private UUID id = UUID.randomUUID();
```

@mlotysz what is your suggestion? We should throw any exception when POST or PUT body has id property?